portfolio/src/main/webapp/script.js

```diff
@@ -16,15 +16,15 @@
  * Adds a random fact to the page.
  */
 function addRandomFact() {
-  const facts =
-      ['I can juggle', 'I know how to unicycle', 'I did the triple jump in high school', 'My major is Civil Engineering'];
-
-  var imgs = new Array ("images/juggling.png", "images/unicycle.png", "images/triplejump.png", "images/civil.png");
+  const facts = [
+    {text: 'I can juggle', imageSrc: 'images/juggling.png'}, {text: 'I know how to unicycle', imageSrc: 'images/unicycle.png'}, 
```

Comment:
Small nitpick: please move the second tuple to a separate line

Reply:
Got it, thanks for the feedback!